backend/forecastbox/api/execution.py

```diff
@@ -85,7 +85,7 @@ def _execute_cascade(spec: ExecutionSpecification) -> tuple[api.SubmitJobRespons
     try:
         job, job_envvars = execution_specification_to_cascade(spec)
     except Exception as e:
-        return api.SubmitJobResponse(job_id=None, error=repr(e)), []
+        return api.SubmitJobResponse(job_id=None, error=str(e)), []
```

Comment:
why is this being changed? `str(e)` is usually less informative than `repr(e)`, in particular the class gets lost